### DISCUSSION TITLE
Editable Filter

### DISCUSSION THREAD

**Discussion by 1ostman** [artemvlas/veretino#6] [Ideas]
Is it planned to implement the ability to edit filtering rules for previously created lists (db)?

I have already created many Veretino databases in my folders. But the contents of the folders can change, files of other types can be added. I would like to be able to adapt the filtering rules to the new content.

**Comment by artemvlas** [comment]
Yes, of course it is planned! But probably in the rather distant future.

**Reply by 1ostman** [reply]
In any case, thank you very much for your efforts! I look forward to future versions!
And at the moment I cope with the above task by simply editing the filter line in the database header... And if you add native support out of the box, it will be absolutely great.
